test/e2e/specs/infrastructure/infrastructure__ssr-enabled.ts

```diff
@@ -15,13 +15,29 @@ describe( DataHelper.createSuiteTitle( 'Server-side Rendering' ), function () {
 		page = await browser.newPage();
 	} );
 
-	it.each`
-		endpoint
-		${ DataHelper.getCalypsoURL( 'log-in' ) }
-		${ DataHelper.getCalypsoURL( 'themes' ) }
-		${ DataHelper.getCalypsoURL( 'theme/twentytwenty' ) }
-	`( 'Check endpoint: $endpoint', async function ( { endpoint } ) {
-		await page.goto( endpoint );
+	// it.each`
+	// 	endpoint
+	// 	${ DataHelper.getCalypsoURL( 'log-in' ) }
+	// 	${ DataHelper.getCalypsoURL( 'themes' ) }
+	// 	${ DataHelper.getCalypsoURL( 'theme/twentytwenty' ) }
+	// `( 'Check endpoint: $endpoint', async function ( { endpoint } ) {
+	// 	await page.goto( endpoint );
+	// 	await page.waitForSelector( '#wpcom[data-calypso-ssr="true"]' );
+	// } );
```

Comment:
**question:blocking**: is there a reason why all tests here are disabled?

Reply:
I'm trying to figure out why they are failing, and I wasn't sure which URL was failing. I don't want to change anything in this test.

Reply:
> I'm trying to figure out why they are failing

Gotcha. To check what tests are failing, you can click on the Details, which links to the TeamCity build in question.
![image](https://user-images.githubusercontent.com/6549265/199565561-8963f1ee-7878-4990-a6d5-984460af0dbd.png)

If required, log in as Guest, or create an account with your A8C email. 
TeamCity will show which specific test failed. In this case, it seems to be the `/theme/twentytwenty` test.

![image](https://user-images.githubusercontent.com/6549265/199565761-990450c0-3da1-45f6-965a-4470b4d10879.png)

Reply:
FYI, a test should not be disabled in order to make the build pass. These tests are failing due to some changes in this PR, as they are passing as expected on `trunk` and other branches.

Reply:
Hey! If you look a little below this line, you'll notice that I extracted the logic from the `it.each`, hoping to get more details. I was doing this here because I couldn't run the e2e tests locally, but now I have managed to run them. I also reverted the changes in this file :) 

I noticed that test is failing because the attribute `data-calypso-ssr` is being removed, and the reason seems to be the `CalypsoShoppingCartProvider`.

Reply:
> I couldn't run the e2e tests locally, but now I have managed to run them

Great to hear you were able to set it up locally! We do have a pretty robust set of document over at [DevDocs](https://wpcalypso.wordpress.com/devdocs/test/e2e/README.md) in case you need to reference things about E2E testing in the future. There are also documents for other levels of testing as well.